src/get-started/java-binding.md

```diff
@@ -503,6 +503,13 @@ Currently Java Binding includes the following built-in services:
   See the [*Anchoring Service*](../advanced/bitcoin-anchoring.md)
   article for more details.
 
+- **Time Oracle.**
+  Time oracle allows determining time, importing it from the external world to
+  the blockchain and keeping its current value in the blockchain.
+
```

Comment:
Do paragraphs in lists work? Would you build this locally and check please, as the published documentation is broken now.

Reply:
Served it locally - it works well.